kubernetes/samples/charts/voyager/README.md

```diff
@@ -106,20 +128,26 @@ Then deploy the TLS Ingress.
 ```
 $ kubectl create -f samples/tls.yaml
 ```
-Now you can access the two WLS domains with different hostnames using the HTTPS endpoint.
+Now you can access the two WebLogic domains with different host names using the HTTPS endpoint.
 ```
 $ curl -k -H 'host: domain1.org' https://${HOSTNAME}:30305/testwebapp/
 $ curl -k -H 'host: domain2.org' https://${HOSTNAME}:30307/testwebapp/
 ```
 
 ## Uninstall the Voyager Operator
 After removing all the Voyager Ingress resources, uninstall the Voyager operator:
+
+Using Helm 3:
+```
+$ helm uninstall voyager-operator --namespace voyager
+```
+Using Helm 2:
 ```
 $ helm delete --purge voyager-operator
 ```
 
 ## Install and uninstall the Voyager operator with setup.sh
-Alternatively, you can run the helper script `setup.sh`, under the `kubernetes/samples/charts/util` folder, to install and uninstall Voyager.
+Alternatively, you can run the helper script `setup.sh` when using Helm 2, under the `kubernetes/samples/charts/util` folder, to install and uninstall Voyager.
```

Comment:
Pani, you should update the text "when using Helm 2" -> "when using Helm 2 or Helm 3" as your updated to `setup.sh` works with Helm 2 or Helm 3. 

It would also be fine to remove this additional text in whole if you prefer....

This text was added for PR#1406 since the script on `master` only works with Helm 2.

Reply:
setup.sh automatically take care both Helm Version, so no need to mention about Helm Version specific usage.  
Additionally  now setup.sh take an optional 3rd parameter to download  specific version of Voyager. Let me update the PR